padd.sh

```diff
@@ -118,7 +121,7 @@ pihole_logo_script_retro_3="${green_text}'  ${red_text}'   ${magenta_text}' ${ye
 
 GetFTLData() {
     local ftl_port LINE
-    ftl_port=$(cat /run/pihole-FTL.port 2> /dev/null)
+    ftl_port=$(docker exec pihole cat /run/pihole-FTL.port 2> /dev/null)
```

Comment:
And if the user does not have docker....?

Reply:
:/ sorry, that should also be configured above, but I'm too stupid to do that :D

Reply:
@phenomeus Take a sneak preview on https://github.com/yubiuser/pihole_adlist_tool/blob/master/pihole_adlist_tool#L238-L252 to **determine if running in docker** or not.

Then you could do this:
```
if [ "$PIHOLE_DOCKER" = 0 ];
    then
        ftl_port=$(cat /run/pihole-FTL.port 2> /dev/null)
    else
        ftl_port=$(docker exec pihole cat /run/pihole-FTL.port 2> /dev/null)

fi
```

But what is this useful for? I think, it's better and easier to **run `padd.sh` within the docker container** as I mentioned in [PR #155](https://github.com/pi-hole/PADD/pull/155#issuecomment-1045327841).